src/interfaces.ts

```diff
@@ -155,6 +155,7 @@ export interface ICoreKit {
   /**
    * The function used to initailise the state of MPCCoreKit
    * Also is useful to resume an existing session.
+   * @param params - handleRedirectResult: boolean - flag for handle the redirect result - Default is true.
    */
   init(): Promise<void>;
```

Comment:
You are adding a parameter to the function doc without adding the parameter to the interface...